solana/program_v2/programs/gateway_v2/src/state/network.rs

```diff
@@ -6,50 +6,71 @@ use anchor_lang::{AnchorDeserialize, AnchorSerialize};
 use bitflags::bitflags;
 
 /// A gatekeeper network which manages many [`Gatekeeper`]s.
-#[derive(Debug)]
 #[account]
+#[derive(Debug)]
 pub struct GatekeeperNetwork {
     /// The version of this struct, should be 0 until a new version is released
     pub version: u8,
     /// The initial authority key
-    pub initial_authority: Pubkey,
-    // TODO: Add Network Features
-    /// Features on the network, index relates to which feature it is. There are 32 bytes of data available for each feature.
-    // pub network_features: Vec<[u8; 32]>,
-    /// The number of auth keys needed to change the `auth_keys`
-    pub auth_threshold: u8,
+    pub authority: Pubkey,
+    /// the index of the network
+    pub network_index: u16,
+    /// The bump for the signer
+    pub network_bump: u8,
     /// The length of time a pass lasts in seconds. `0` means does not expire.
     pub pass_expire_time: i64,
-    /// The bump for the signer
-    pub signer_bump: u8,
+    /// Features on the network, index relates to which feature it is. There are 32 bytes of data available for each feature.
+    pub network_features: Vec<[u8; 32]>,
```

Comment:
Check the doc, this will now just be a u32.

The data stored in `network_features_data` (commented out)

Reply:
Ah whoops... Will fix...

Reply:
resolved